### PR TITLE
Add the Question in JS Guess the Sum of expression?

### PR DESCRIPTION
**Pull request by Shaik-mohd-huzaifa**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -1837,6 +1837,16 @@ const javascriptQuiz = [
     Explanation:
       "JavaScript has 7 primitive data types. Boolean, null, undefined, Number, BigInt, String and Symbol",
     Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_Types#data_structures_and_types"
+  },
+  {
+    Question: "What is the Sum of this Expression (true + 34)",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:1842]:
```suggestion
    Question: "In JavaScript, what is the result for the following code: console.log(true + 34)?",
```

File: src/data/javascript-quiz.ts [hunk 2]
```diff
@@ -1837,6 +1837,16 @@ const javascriptQuiz = [
...
+    Answer: "35",
+    Distractor1: "34",
+    Distractor2: "true34",
+    Distractor3: "35",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:1846]: distractors are for wrong answers only

```suggestion
    Distractor3: "25",
```

File: src/data/javascript-quiz.ts [hunk 3]
```diff
@@ -1837,6 +1837,16 @@ const javascriptQuiz = [
...
+    Explanation:
+      "Here the True gets converted to a number and gives out 1. Because 0 = false and 1 = true",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:1848]:
```suggestion
      "Since true is converted to the number 1, then true + 34 would result in 35.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by Shaik-mohd-huzaifa — COMMENTED** [review]
Changes are done now you can accept the pull request

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**